README.md

```diff
@@ -1,144 +1,176 @@
 # Stackanetes
 
+Stackanetes is an initiative to make operating OpenStack as simple as running any application on Kubernetes.
+Stackanetes deploys standard OpenStack services into containers and uses Kubernetes’ robust application lifecycle management capabilities to deliver a single platform for companies to run OpenStack Infrastructure-as-a-Service (IaaS) and container workloads.
+
 ## Overview
 
-Stackanetes is heavily based on kolla-mesos (project abandoned).
-Link to base [kolla repo](https://github.com/openstack/kolla)
+### Services
+
+Stackanetes sets up the following OpenStack components:
+- Cinder
+- Glance
+- Horizon
+- Keystone
+- Neutron
+- Nova
+- Searchlight
+
+In addition to these, a few other applications are deployed:
+- MariaDB
+- Memcached
+- RabbitMQ
+- RADOS Gateway
+- Traefik
+- Elasticsearch
+- Open vSwitch
+
+Services are divided and scheduled into two groups, with the exception of the Open vSwitch agents which run everywhere:
+- The control plane, which runs all the OpenStack APIs and every other supporting applications,
+- The compute plane, which is dedicated to run Nova's virtual machines.
+
+### Gotta go fast
+
+Leaving aside the configuration of the [requirements], Stackanetes can fully deploy OpenStack from scratch in ~5-8min. But that's not the only strength of Stackanetes, it's true power resides in its ability to help managing OpenStack's lifecycle. 
+
+[requirements]: #requirements
+
+### Requirements
+
+Stackanetes requires Kubernetes 1.3+ with:
+  - At least two schedulable nodes,
+  - At least one virtualization-ready node,
+  - [Overlay network] & [DNS add-on],
+  - Kubelet running with `--allow-privileged=true`,
+
+While Glance may operate with local storage, a Ceph cluster is needed for Cinder.
+Nova's live-migration feature also requires Ceph and DNS resolution of the Kubernetes nodes' hostnames.
```

Comment:
CEPH is not necessary for live migration, but it's required for full HA (ie: for glance).